rerun_cpp/src/rerun/components/rotation3d_ext.cpp

```diff
@@ -29,6 +29,6 @@ namespace rerun {
 #define Rotation3DExt Rotation3D
 #endif
 
-        const Rotation3D Rotation3D::IDENTITY = datatypes::Quaternion::IDENTITY;
+        const Rotation3DExt Rotation3DExt::IDENTITY = datatypes::Quaternion::IDENTITY;
```

Comment:
I hadn't noticed before we had the define Rotation3DExt Rotation3D above. I think this is actually the right pattern.